packages/loader/driver-utils/src/treeConversions.ts

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Uint8ArrayToString,unreachableCase} from "@fluidframework/common-utils";
+import { AttachmentTreeEntry, BlobTreeEntry, TreeTreeEntry } from "@fluidframework/protocol-base";
+import * as api from "@fluidframework/protocol-definitions";
+
+/**
+ * Converts ISummaryTree to ITree format.
+ * @param summaryTree - summary tree in ISummaryTree format
+ */
+ export function convertSummaryTreeToSnapshotITree( summaryTree: api.ISummaryTree ): api.ITree {
```

Comment:
```suggestion
 export function convertSummaryTreeToSnapshotITree(summaryTree: api.ISummaryTree): api.ITree {
```
nit: remove whitespace